### PR TITLE
Changing drop down for preference in app table to text only

### PR DESCRIPTION
**Pull request by amyleblond**
Changes in PR:

**models.py**: changing the field type for preference to be a drop down
**style.css**: changing the width of the output to be large enough to show all text
**application.html**: removing the numbers from the legend so we dont confuse people

### CODE REVIEW

**Comment by amyleblond** [comment]
screen shot 2018-02-07 at 12 47 40 pm

File: taworks/taform/models.py [hunk 1]
```diff
@@ -60,7 +60,8 @@ class Application(models.Model):
     student = models.ForeignKey(Student, on_delete=models.CASCADE, blank=True)
     course = models.ForeignKey(Course, on_delete=models.CASCADE, blank=True)
     application_date = models.DateTimeField(auto_now_add=True)
-    preference = models.IntegerField(validators=[MaxValueValidator(3), MinValueValidator(0)])
+    STUDENT_PREFERENCE = ((1,'I prefer to TA this course'),(2,'I am able to TA this course'),(3,'I would prefer not to TA this course'),(0,'I am unable to TA this course'))
+    preference = models.CharField(max_length=50, choices=STUDENT_PREFERENCE)
```
Comment (j2kan) [inline taworks/taform/models.py]: i think in the database we still want to store it as integerField for the algorithm.

File: taworks/taform/templates/taform/application.html [hunk 2]
```diff
@@ -124,12 +124,12 @@ <h3>Course Application</h3>
...
     schedule for each course before applying. <p>
 
-    <p>For each course, please enter your preference. See the legend below for details:</p>
+    <p>For each course, please enter your preference. See the options below for details:</p>
     <ul>
-        <li> <b>1:</b> I prefer to teach this course</li>
-        <li> <b>2:</b> I am able to teach this course</li>
-        <li> <b>3:</b> I would prefer not to teach this course</li>
-        <li> <b>0:</b> I am unable to TA this course</li>
+        <li>I prefer to teach this course</li>
```
Comment (j2kan) [inline taworks/taform/templates/taform/application.html]: these needs to line up with the drop down msg.

**Comment by amyleblond** [comment]
screen shot 2018-02-07 at 2 25 57 pm
Saves as INT!

**Review by j2kan — APPROVED** [review]
lgtm

**Pull request merged**